core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java

```diff
@@ -708,4 +692,45 @@ public void recoverFromStartSync(AlluxioURI uri, long mountId) {
       future.cancel(true);
     }
   }
+
+  @Override
+  public boolean processJournalEntry(JournalEntry entry) {
+    if (entry.hasAddSyncPoint()) {
+      apply(entry.getAddSyncPoint());
+      return true;
+    } else if (entry.hasRemoveSyncPoint()) {
+      apply(entry.getRemoveSyncPoint());
+      return true;
+    } else if (entry.hasActiveSyncTxId()) {
+      File.ActiveSyncTxIdEntry activeSyncTxId = entry.getActiveSyncTxId();
+      setTxId(activeSyncTxId.getMountId(), activeSyncTxId.getTxId());
+      return true;
+    }
```

Comment:
Any good way to guarantee we have all the cases?

Reply:
same as before, unrecognized entries will cause a runtime exception to be thrown